src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
```

Comment:
I'm still not happy with the function description. The wrap is not specializing on determining the menu width.